scan.go

```diff
@@ -134,6 +135,8 @@ func newScanCommand() *cobra.Command {
 		"Github organization URL (Example: https://github.com/safedep)")
 	cmd.Flags().IntVarP(&githubOrgMaxRepositories, "github-org-max-repo", "", 1000,
 		"Maximum number of repositories to process for the Github Org")
+	cmd.Flags().StringVar(&githubOrgExcludeRepos, "github-org-exclude-repos", "", 
```

Comment:
This should be a `StringArrayVarP` so that cobra automatically creates the array and accepts multiple `--github-org-exclude-repos`

See example: https://github.com/safedep/vet/blob/main/scan.go#L129